wagtail/wagtailcore/tests/test_dbwhitelister.py

```diff
@@ -15,45 +15,65 @@ def assertHtmlEqual(self, str1, str2):
         self.assertEqual(BeautifulSoup(str1, 'html5lib'), BeautifulSoup(str2, 'html5lib'))
 
     def test_page_link_is_rewritten(self):
-        input_html = '<p>Look at the <a data-linktype="page" data-id="2" href="/">lovely homepage</a> of my <a href="http://wagtail.io/">Wagtail</a> site</p>'
+        input_html = '<p>Look at the <a data-linktype="page" data-id="2" href="/">lovely homepage</a>'
+        ' of my <a href="http://wagtail.io/">Wagtail</a> site</p>'
```

Comment:
Ok, I'll make the corrections. Thank you.

Maris

On 7 December 2015 at 22:00, Matt Westcott <notifications@github.com> wrote:

> In wagtail/wagtailcore/tests/test_dbwhitelister.py
> <https://github.com/torchbox/wagtail/pull/2004#discussion_r46884271>:
>
> > @@ -15,45 +15,65 @@ def assertHtmlEqual(self, str1, str2):
> >          self.assertEqual(BeautifulSoup(str1, 'html5lib'), BeautifulSoup(str2, 'html5lib'))
> >
> >      def test_page_link_is_rewritten(self):
> > -        input_html = '<p>Look at the <a data-linktype="page" data-id="2" href="/">lovely homepage</a> of my <a href="http://wagtail.io/">Wagtail</a> site</p>'
> > +        input_html = '<p>Look at the <a data-linktype="page" data-id="2" href="/">lovely homepage</a>'
> > +        ' of my <a href="http://wagtail.io/">Wagtail</a> site</p>'
>
> Splitting a string like this won't work, because Python sees the first
> line as a completely valid line in its own right. You could fix this by
> adding brackets:
>
>     input_html = ('<p>Look at the <a data-linktype="page" data-id="2" href="/">lovely homepage</a>'
>         ' of my <a href="http://wagtail.io/">Wagtail</a> site</p>')
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/torchbox/wagtail/pull/2004/files#r46884271>.
>